pkg/config/types.go

```diff
@@ -28,6 +34,7 @@ type UserConfig struct {
 	Bucket           string `json:"bucket"`
 	CustomScriptPath string `json:"custom-script"`
 	ConfigFilePath   string `json:"config-file"`
+	IsDemo           bool   `json:"demo-mode"`
```

Comment:
similar to integ test check-- `--demo` should check that the DB exists before kicking everything off. However, this can be addressed when I refactor to front load all validation